app/components/unverified-user-message.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+
+  isMessageVisible: true,
+
+  showMessage: computed('session.isAuthenticated', 'authManager.currentUser.isVerified', 'isMessageVisible', function() {
+    let isAuthenticated = this.get('session.isAuthenticated');
```

Comment:
Why the unnecessary variables and the redundant conditional. ?

You could just do the below right ?

```js
return this.get('isMessageVisible') 
    && this.get('session.isAuthenticated') 
    && !this.get('authManager.currentUser.isVerified');
```